src/main/java/org/syftkog/web/test/framework/Language.java

```diff
@@ -0,0 +1,191 @@
+package org.syftkog.web.test.framework;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import javax.xml.bind.annotation.XmlEnum;
+import javax.xml.bind.annotation.XmlType;
+
+/**
+ *
+ * @author BenjaminLimb
+ */
+@XmlType(name = "langauge")
+@XmlEnum
+public enum Language {
+
+  /**
+   *
+   */
+  DE("Deutsch", "German", "de", "deu"),
+
+  /**
+   *
+   */
+  PT("Português", "Portuguese", "pt", "por"),
+
+  /**
+   *
+   */
+  EN("English", "English", "en", "eng"),
+
+  /**
+   *
+   */
+  RU("Русский", "Russian", "ru", "rus"),
+
+  /**
+   *
+   */
+  ES("Español", "Spanish", "es", "spa"),

```

Comment:
I'm not about other languages on this list but Spanish does not capitalize names of languages...